### DISCUSSION TITLE
Questions from a new Lay-Z-Spa Module user

### DISCUSSION THREAD

**Discussion by WesWesthaver** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#605] [Q&A]
I have a "SaluSpa Paris" hot tub with an Egg shaped pump Model No. 54149E. 

[Picture of my pump's ID label](http://www.westhaver.com/SaluSpa/IMG_20220924_173721.jpg)

I purchased a pre-built Lay-Z-Spa Module kit from another user here (thanks metaljay) and have installed it. 

[Picture of my Lay-Z-Spa Module](http://www.westhaver.com/SaluSpa/IMG_20230822_193805.jpg)

I was able to connect to it with my cell phone and get the WiFi configured. I set up a static lease on my router so that the Lay-Z-Spa Module always gets the same IP address. So the network communication is working fine.

I set the "Hardware Config" to my pump and display model (ie: "6 wire, 54149E (air)") and PCB to V2B. But I had no control of my spa. So I tried other pump/display options and the only one that worked for me was "6 wire, pre 2021 (air)." Why doesn't my pump work when configured to the actual pump model?

The only other peculiarity that I have noticed is that when I press a control panel button it makes what sounds like a computer generated sound rather than the original simple beep sound. But it is so quiet that it is barely audible. What can I do to increase the volume level of the new sound?

I am really excited about getting familiar with this system. Thanks visualapproach!

-Wes

**Comment by visualapproach** [comment]
Hi, regarding the model name, I don't know but BW has a lot of model names and they are not always consistent...
The audio is generated in the ESP and some models has a different audio hardware and sound will be more quiet. Don't know what to do other than:
Find out what circuit is needed. Or mount a separate buzzer. Or just connect audio from pump to display without involving the ESP
